src/js/bs3/module/HintPopover.js

```diff
@@ -0,0 +1,273 @@
+define([
+  'summernote/base/core/func',
+  'summernote/base/core/list',
+  'summernote/base/core/dom',
+  'summernote/base/core/range',
+  'summernote/base/core/key'
+], function (func, list, dom, range, key) {
+  var HintPopover = function (summernote) {
+    var self = this;
+    var ui = $.summernote.ui;
+
+    var $note = summernote.layoutInfo.note;
+    var hint = summernote.options.hint || false;
+
+    if (hint && !(hint instanceof Array)) {
+      hint = [hint];
+    }
+
+    var KEY = key.code;
+
+    var DROPDOWN_KEYCODES = [KEY.UP, KEY.DOWN, KEY.ENTER];
+
+    var $popover = ui.popover({
+      className: 'note-hint-popover',
+      callback : function ($node) {
+        $node.css({
+          'min-width': '100px',
+          'padding': '2px'
+        });
+      }
```

Comment:
We have a `.note-hint-popover` selector on `summernote.less`. Do you have any reason for style code here?